### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by ivailo1st**
Added extra information about the new widget configuration settings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ Widgets are defined by the following fields (and can furthermore be imported/exp
 
 -   **Name** The displayname to show inside Colibo for widget-editors
 -   **Description** A short description for the editor to choose the right widget.
+-   **Categories** A list of texts used for defining the different categories a widget is apart of
```
Comment (FlemmingRavn) [inline README.md]: Maybe give a few examples of categories

File: README.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ Widgets are defined by the following fields (and can furthermore be imported/exp
...
+-   **PosterImage** A url used for linking to an image that is displayed as a poster of the widget in the widget library
+-   **ExampleImages** A list of urls that are used for linking images that are displayed as examples of how a widget looks like when its being used
```
Comment (FlemmingRavn) [inline README.md]:
```suggestion
-   **ExampleImages** A list of urls to images that are displayed as examples of how a widget looks like when its being used
```

File: README.md [hunk 3]
```diff
@@ -20,6 +20,9 @@ Widgets are defined by the following fields (and can furthermore be imported/exp
 
 -   **Name** The displayname to show inside Colibo for widget-editors
 -   **Description** A short description for the editor to choose the right widget.
+-   **Categories** A list of texts used for defining the different categories a widget is apart of
+-   **PosterImage** A url used for linking to an image that is displayed as a poster of the widget in the widget library
```
Comment (FlemmingRavn) [inline README.md]:
```suggestion
-   **PosterImage** A url to an image that is displayed as a poster of the widget in the widget library
```

File: README.md (hunk 1 above)
Comment (ivailo1st) [reply README.md]: Added examples of categories.

File: README.md (hunk 2 above)
Comment (ivailo1st) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (ivailo1st) [reply README.md]: Done

**Review by FlemmingRavn — APPROVED** [review]

**Pull request merged**